charts/harvester-cloud-provider/100.0.0+up0.1.8/Chart.yaml

```diff
@@ -1,10 +1,10 @@
 annotations:
   catalog.cattle.io/certified: rancher
   catalog.cattle.io/display-name: Harvester Cloud Provider
-  catalog.cattle.io/kube-version: '>= 1.18'
+  catalog.cattle.io/kube-version: '>= 1.18.0-0 < 1.22.0-0'
```

Comment:
is this just for the support matrix doc? currently, both Harvester CSI and CCM chart are verified with k8s version from v1.21.x to v1.24.x, so should we change this one accordingly?

Reply:
@guangbochen This PR is only modifying released versions of the harvester charts in the release-v2.6 branch so it won't affect the other PR you linked. I just need verification that the kube-version constraints I added to those older versions are correct. Would it make sense to set all those older version to use `< 1.24.0-0'` instead instead of `< 1.22.0-0'`?

Reply:
Yes, using kube-version with `< 1.24.0-0` will be more appropriate, thanks. cc @yaocw2020

Reply:
@guangbochen yes, we are enforcing a closed range so that we can eventually automate generating a support matrix. This is only for older version of the charts as this change is made to the release-v2.6 branch and not dev, so it won't affect the PR you linked. Should I update the constraint of all these old charts to < 1.25 instead of < 1.22?

Reply:
@guangbochen Just updated, can you re-review?